parts/kuberneteswindowssetup.ps1

```diff
@@ -201,8 +201,17 @@ Get-PodGateway(`$podCIDR)
 function
 Set-DockerNetwork(`$podCIDR)
 {
-    # Turn off Firewall to enable pods to talk to service endpoints. (Kubelet should eventually do this)
-    netsh advfirewall set allprofiles state off
+    # Windows Firewall rules to allow only Master to access Node's kubelet ports
+    # Firewall rules to allow access to container's websockets
+    netsh advfirewall firewall add rule name="Container: Allow access to node localport 8080" dir=in action=allow protocol=TCP localport=8080
+    netsh advfirewall firewall add rule name="Container: Allow access to node localport 8888" dir=in action=allow protocol=TCP localport=8888
+    netsh advfirewall firewall add rule name="Container: Allow UDP inbound traffic for Container DNS Port 53" dir=in action=allow localport=53 protocol=UDP
+    netsh advfirewall firewall add rule name="Node: Allow only K8 Master to access localport 4194" dir=in action=allow protocol=TCP localport=4194 remoteip=`${global:MasterIP}

```

Comment:
@robbiezhang To test the firewall rules, I used the Master's well known private IP [10.240.255.5], since this is a well known acs-engine constant.
So yes, this is internal.

Looking at the script [line 190], the global variable is set to the script parameter value.